src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -212,11 +212,11 @@ class Client extends BaseTransport implements QueryTransport, WritingInterface,
     /**
      * @var string
      */
-    private $caseSensitiveEncoding;
+    private $caseSensitiveEncoding = null;
 
     /**
      * @param FactoryInterface $factory
-     * @param Connection       $conn
+     * @param Connection $conn
```

Comment:
these are usually aligned. we seem to not have a codestyle thing that complains. as the parameters are declared in the method signature anyways and there is no additional information, we could also simply delete the annotations.